src/gui/src/views/admin/WordListsView.vue

```diff
@@ -14,7 +14,11 @@
       @update-items="updateData"
```

Comment:
After edit/update: No worldlist entries are found anymore

Reply:
Couldn't reproduce, maybe was already fixed at the point I read it.